ci-operator/step-registry/openstack/conf/createbootstrapfip/openstack-conf-createbootstrapfip-command.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml
+CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
```

Comment:
You'll have `"${SHARED_DIR}"/CLUSTER_NAME` if you run `openstack-conf-resourcenames`

Reply:
done